### DISCUSSION TITLE
How to download portable version

### DISCUSSION THREAD

**Discussion by ppazos** [nroduit/Weasis#516] [General]
Hi all, some time ago I was able to download portable versions of Weasis and now it seems it's not possible, the only ways available to download Weasis are via installation packages and external installation tools. 
Is thee a way that I can download a portable version without an installer? Thanks.

PS: I think previously there were AppImage downloads available.

**Comment by nroduit** [comment]
No, see this [post](https://groups.google.com/g/dcm4che/c/VGk7ziWPvp8/m/DAQMZx14BQAJ) and https://github.com/nroduit/Weasis/discussions/418

**Reply by ppazos** [reply]
@nroduit is there any documentation on how it was packaged before the portable was removed? Thanks
